cddl/Makefile

```diff
@@ -57,3 +58,9 @@ clean: ; rm -f $(CLEANFILES)
 
 # Extract the CBOR tags defined by CoRIM/CoMID (i.e., those in the 5xx space)
 cbor-tags.txt: $(wildcard *.cddl) ; grep -h '#6\.5' *cddl | sort -u -t'=' -k2 > $@
+
+# Return an error if there are duplicat CBOR Tags defined
+cbor-tags-unique: $(wildcard *.cddl) ; \
+	 grep -h '#6\.5' *cddl | sort -u -t'=' -k2 > cbor-tags.txt ; \
+	 grep -E -h -o '#6\.[0-9]{3}' cbor-tags.txt | uniq -cd ; \
+	 if [ ! -z $1 ]; then exit 1; else echo "CBOR tags are unique"; fi
```

Comment:
`$1` is a positional parameter, I don't think this will expand to anything useful.